BorrowMyProducts/src/main/java/com/techhunters/borrowmyproducts/service/AuthService.java

```diff
@@ -1,4 +1,17 @@
 package com.techhunters.borrowmyproducts.service;
 
-public interface AuthService {
+
+public interface AuthService 
+{
+	//By Default the parameters are public static final 
+	 String ACCOUNT_SID = "ACa95f20b59d30fc54a834d259c85df35d";
```

Comment:
never put keys and password in java code. Put them is application.properties and fetch from there